packages/server/src/modules/documents/helpers/document-type.helper.ts

```diff
@@ -0,0 +1,27 @@
+import { DocumentType } from '@shared/enums';
```

Comment:
I see we've got duplicated enum (one hard coded and one generated from schema). Let's stick with the generated one, I'll get rid of the hard coded in a different PR

```suggestion
import { DocumentType } from '@shared/gql-types';
```